.github/workflows/smoketest.yml

```diff
@@ -61,3 +61,23 @@ jobs:
         run: npm install
       - name: Lint
         run: npm run-script lint
+  build-test:
+    runs-on: ubuntu-latest
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      - name: Use Node.js 12
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - name: Cache npm local files
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.npm
+            ./frontend/node_modules
+          key: npm-${{ hashFiles('frontend/package-lock.json', 'frontend/package.json') }}
+      - name: Node setup
+        run: npm install
+      - name: Build release
+        run: npm run-script build
```

Comment:
random thought: long run, is there an ability for this to test the presence of certain env variables in the build, or if those variables have the right values?

Reply:
Totally! In that we can run arbitrary code and just have it fail the build by returning a non-zero status. 😁